### DISCUSSION TITLE
Changing dashboard base image from nginx to nginx-unpriviledged

### DISCUSSION THREAD

**Discussion by mattelen** [sorry-cypress/sorry-cypress#684] [Ideas]
I'm keen to seek feedback on the thought of changing the dashboard base image to using nginx-unpriviledged (https://hub.docker.com/r/nginxinc/nginx-unprivileged).

I'm keen for this change because on some restrictive k8s deployments, like Openshift, the basic nginx image with a config like Sorry Cypress is not able to be deployed. As the current image starts by running the container on port 80, and then flips to 8080 if the config requires, Openshift does not allow access to port 80, so it halts the process. 

I did a quick test, and the default of running on port 80 still works for non k8s, like for local development.
I'm also aware that this issue we have might be super specific to our requirements, hence why I'm keen for feedback on if this change would help benefit the wider community of Sorry Cypress, or not. TIA.

**Comment by agoldis** [comment]
I think it would be a big deal if it works out-of-the-box and the transition is easy or seamless. We can also have a separate dockerfile / image using nginx-unpriviledged - would that work for you?

**Reply by mattelen** [reply]
I think the separate image that can be used from docker hub might be the best way, as I'm sure we don't want to upset how people currently deploy their images, but at the same time, I'm aware of the extra overhead this may bring. Otherwise, if it is too much extra overhead, then we could do further, more directed testing to ensure that this change would work out of the box and the transition is easy or seamless

**Reply by agoldis** [reply]
Would you be willing to create a PR? I will follow up with publishing

**Reply by mathpaquette** [reply]
@mattelen ping me if you need help.

**Reply by mattelen** [reply]
Thanks team. I'll ping you both on the PR to check it out!

**Comment by mathpaquette** [comment]
@agoldis I think this can be seamless. Let me give a try
